madminer/utils/morphing.py

```diff
@@ -113,7 +120,7 @@ def set_components(self, components):
         self.components = components
         self.n_components = len(self.components)
 
-    def find_components(self, max_overall_power=4):
+    def find_components(self, max_overall_power=4, BSM_max_power=float("inf"), Nd=0, Np=0, Ns=0):
```

Comment:
Suggestion: I would find an order where `Nd` appears before `Np` more intuitive, as production happens before decay. It does not matter for the code, and users can specify the keyword arguments in the order they like, so I won't insist on this change.

Reply:
Sure, I think I have changed all orders into Np, Nd, Ns in parameter inputs and comments in the new commit.